docs/3.0.0-beta.x/getting-started/quick-start.md

```diff
@@ -1,118 +1,117 @@
 # Quick Start Guide
 
-Get ready to get Strapi up and running in **less than 5 minutes** 🚀
+Get ready to get Strapi up and running in **less than 5 minutes** 🚀.
 
 <div class="video-container">
 <iframe width="853" height="480"  src="https://www.youtube.com/embed/_qlLobVjd9k" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
 _For a step-by-step guide, please take a look at the [detailed tutorial](quick-start-tutorial.html)._
 
-## 1. Install Strapi globally
+(Before continuing, please make sure [Node.js and npm are properly installed](install-requirements.html) on your machine.)
 
-Please make sure [Node.js and npm are properly installed](install-requirements.html) on your machine.
+## 1. Install Strapi and Create a new project
 
 ```bash
-npm install strapi@beta -g
+yarn create strapi-app my-project --quickstart
```

Comment:
You should link the yarn installation doc like your are doing with node and npm

Reply:
Done.